src/server/dfly_main.cc

```diff
@@ -3,13 +3,6 @@
 // See LICENSE for licensing terms.
 //
 
-#include "absl/cleanup/cleanup.h"
-#include "absl/container/inlined_vector.h"
-#include "absl/strings/numbers.h"
-#ifdef NDEBUG
```

Comment:
so we now always allocate via mimalloc. worth adding this to commit description as well.